src/vg/civcraft/mc/civmodcore/Config.java

```diff
@@ -1,15 +1,13 @@
 package vg.civcraft.mc.civmodcore;
 
-import java.io.IOException;
+import static vg.civcraft.mc.civmodcore.CivModCorePlugin.log;
```

Comment:
Plugins extend ACivMod, not CivModCorePlugin. Should import ACivMod's log, not the subclass CivModCorePlugin, unless I'm misunderstanding your intent here.